src/index.js

```diff
@@ -188,26 +120,25 @@ const Clap = class extends React.Component {
 
   render () {
     const {count, countTotal, isClicked} = this.state
-    const {size, primaryColor, secondaryColor, iconComponent: ClapIcon} = this.props
+    const {iconComponent: ClapIcon, theme} = this.props
 
     return (
-      <ClapButton id='clap' primaryColor={primaryColor} secondaryColor={secondaryColor} size={size} onClick={this.onClick}>
-        <ClapIcon id='clap--icon' isClicked={isClicked} primaryColor={primaryColor} secondaryColor={secondaryColor} />
-        <ClapCount id='clap--count' secondaryColor={secondaryColor} size={size}>
-          +{count}
-        </ClapCount>
-        <ClapCountTotal primaryColor={primaryColor} id='clap--count-total' size={size}>
-          +{countTotal}
-        </ClapCountTotal>
-      </ClapButton>
+      <ThemeProvider theme={Object.assign(defaultTheme, theme || {})}>
```

Comment:
I suppose you are not using `defaultTheme` as `defaultProps` to be possible merge props between the data provided and the default styling.

Could be more readable create a function to returns this value. Something like:

```js
const getTheme =  ({theme}) = {{}) => Object.assign({}, defaultTheme, theme)
```